services/usb-livesync-service-base.ts

```diff
@@ -0,0 +1,125 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import helpers = require("./../helpers");
+import minimatch = require("minimatch");
+import path = require("path");
+import util = require("util");
+
+let gaze = require("gaze");
+
+interface IProjectFileInfo {
+	fileName: string;
+	onDeviceName: string;
+	shouldIncludeFile: boolean;
+}
+
+export class UsbLiveSyncServiceBase implements IUsbLiveSyncServiceBase {
+	constructor(protected $devicesServices: Mobile.IDevicesServices,
+		protected $mobileHelper: Mobile.IMobileHelper,
+		private $localToDevicePathDataFactory: Mobile.ILocalToDevicePathDataFactory,
```

Comment:
can you order the dependencies based on their privacy... just move this private dependency next to other private dependencies if it is possible